pySDC/helpers/fieldsIO.py

```diff
@@ -69,11 +69,30 @@ class MPI:
 DTYPES = {
     0: np.float64,  # double precision
     1: np.complex128,
-    2: np.longdouble,  # quadruple precision
-    3: np.clongdouble,
-    4: np.float32,  # single precision
-    5: np.complex64,
 }
+try:
+    DTYPES.update(
+        {
+            2: np.float128,  # quadruple precision
+            3: np.complex256,
+        }
+    )
+except AttributeError:
+    import logging
```

Comment:
Can logging not be a module import ? Local import with black formating makes this very ugly ...